speech-commands/training/browser-fft/training_custom_audio_model_in_python.ipynb

```diff
@@ -0,0 +1,815 @@
+{
```

Comment:
<p>does this assume numpy, tensorflow, scipy, etc are all pre-installed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tfjs-models/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Yes. We can safely assume that these libraries are all pre-installed for the following reasons:</p><p> </p><ol><li>They are installed by default on Google Colab.</li><li>These three are such widely-used libraries that virtually all data scientists and ML practitioners know how to install them. </li></ol>